### PR TITLE
kubelet: use ConfigurationMap instead of json

### PR DESCRIPTION
**Pull request by mikedanese**
Easier to seperate.

### CODE REVIEW

**Comment by mikedanese** [comment]
@k8s-bot unit test this please

File: docs/admin/kubelet.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ kubelet
       --network-plugin="": <Warning: Alpha feature> The name of the network plugin to be invoked for various events in kubelet/pod lifecycle
       --network-plugin-dir="/usr/libexec/kubernetes/kubelet-plugins/net/exec/": <Warning: Alpha feature> The full path of the directory in which to search for network plugins
       --node-ip=<nil>: IP address of the node. If set, kubelet will use this IP address for the node
-      --node-labels="": <Warning: Alpha feature> Labels to add when registering the node in the cluster.  Labels must be specified as a json map of key:value pairs.
+      --node-labels=: <Warning: Alpha feature> Labels to add when registering the node in the cluster.  Labels must are key=value pairs sperated by ','
```
Comment (smarterclayton) [inline docs/admin/kubelet.md]: Typo

Comment (mikedanese) [inline docs/admin/kubelet.md]: Fixed

**Comment by mikedanese** [comment]
I have just realized that the title of my PR may be missleading. ConfigurationMap is not to be confused with ConfigMap.

**Comment by pwittrock** [comment]
Thanks @mikedanese.  This is great.

**Pull request merged**